### PR TITLE
Feat: added /ping API in UI 

### PR DESCRIPTION
**Pull request by DmitriyLewen**
Issue #179

### CODE REVIEW

File: ui/backend/uiserver/server.go [hunk 1]
```diff
@@ -68,3 +71,10 @@ func (srv *uiServer) Start() {
 func (srv *uiServer) Stop() {
 	log.Print("UI Postee server stopped!")
 }
+
+func (ctx *uiServer) pingHandler(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Content-Type", "application/json; charset=UTF-8")
+	w.WriteHeader(http.StatusOK)
+	jsonStr, _ := json.Marshal("Postee UI alive!")
```
Comment (AndreyLevchenko) [inline ui/backend/uiserver/server.go:78]: no need to marshal json. Empty body is fine here

File: ui/backend/uiserver/server.go [hunk 2]
```diff
@@ -68,3 +71,10 @@ func (srv *uiServer) Start() {
 func (srv *uiServer) Stop() {
 	log.Print("UI Postee server stopped!")
 }
+
+func (ctx *uiServer) pingHandler(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Content-Type", "application/json; charset=UTF-8")
```
Comment (AndreyLevchenko) [inline ui/backend/uiserver/server.go:76]: this is not required

**Review by AndreyLevchenko — CHANGES_REQUESTED** [review]

File: ui/backend/uiserver/server.go (hunk 2 above)
Comment (DmitriyLewen) [reply ui/backend/uiserver/server.go]: done

File: ui/backend/uiserver/server.go (hunk 1 above)
Comment (DmitriyLewen) [reply ui/backend/uiserver/server.go]: done

Comment (DmitriyLewen) [reply ui/backend/uiserver/server.go]: Done

File: ui/backend/uiserver/server.go (hunk 2 above)
Comment (DmitriyLewen) [reply ui/backend/uiserver/server.go]: Done

**Review by AndreyLevchenko — APPROVED** [review]

**Review by AndreyLevchenko — APPROVED** [review]

**Pull request merged**